docs/object-protection-testing.md

```diff
@@ -0,0 +1,54 @@
+# Object Integrity testing
```

Comment:
Actually most .md files don't have license block.  For example, [https://github.com/Seagate/cortx-s3server/blob/main/SUPPORT.md.](https://github.com/Seagate/cortx-s3server/blob/main/SUPPORT.md.)  In fact, I don't know any .md files that have license block.  Those are only needed in source code files but not documentation files.

Reply:
@johnbent 

Can we please have this information communication team-wide in an email on some other appropriate way?  otherwise we were instructed to have License block in every .md file and we're trying to follow this, since this was the requirement.